torch/testing/_internal/schema_check_tensor.py

```diff
@@ -1,11 +1,16 @@
 import torch
 from torch.utils._pytree import tree_map
+from torch.fx.operator_schemas import get_signature_for_torch_op
+from torch.testing._internal.jit_utils import clone_inputs
+from torch.testing._internal.common_utils import is_iterable_of_dtype
+from typing import Any, Callable, Dict, Tuple
 
 schema_check_recorded_ops = []
```

Comment:
I feel a bit wary of globals. Can't we just store the sequence of ops on a tensor subclass object?
There's some potential for data races here.